.circleci/testnet-cleanup-mark.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+const path = require('path');
+const fs = require('fs');
+
+const configFilePath = path.join(process.cwd(), 'config.json');
+const { getClosedPullRequests, getBoyarChainConfigurationById, markChainForRemoval } = require('./boyar-lib');
+
+// Read the Boyar config from file
+// We use this const variable as a solid base to start modifying
+const configuration = require(configFilePath);
+
+(async function () {
+    console.log('Querying closed github PRs...');
+    const closedPRs = await getClosedPullRequests();
+
+    let removeCounter = 0;
+    let updatedConfiguration = configuration;
+
+    for (let key in closedPRs) {
+        let aClosedPR = closedPRs[key];
+        let aClosedPRChainId = 100000 + aClosedPR.number;
```

Comment:
I'm assuming this isn't the only place you're calculating chain id from PR number - you need to encapsulate it

Reply:
Refactored